pkg/initializer/validator/datavalidator_test.go

```diff
@@ -136,7 +138,10 @@ var _ = Describe("Running Datavalidator", func() {
 			Expect(err).ShouldNot(HaveOccurred())
 
 			tempFile := path.Join(outputDir, "temp", "db")
-			err = copyFile(dbFile, tempFile)
+
+			fileInfo, err := os.Stat(dbFile)
```

Comment:
We are already running `os.Stat()` on the `dbFile` 3 lines before this. You can simply reuse the output from that to populate `fileInfo`.